pkg/common/telemetry/specific_metrics.go

```diff
@@ -0,0 +1,428 @@
+package telemetry
```

Comment:
Many functions in `telemetry` package start with the same prefix. I think sub-packages would help.

For example how about
```
registrationapitelemetry.UpdateEntryCall
registrationapitelemetry.ListFedBundlesCall
```
or
```
registrationapicall.UpdateEntry
registrationapicall.ListFedBundles
```
instead of 
```
telemetry.RegistrationAPIUpdateEntryCall
telemetry.RegistrationAPIListFedBundlesCall
```